kaolin/graphics/SoftRenderer.py

```diff
@@ -106,36 +125,55 @@ def __init__(
                 the `look` mode, only the direction in which the camera
                 needs to look is specified. It does not necessarily look
                 towards the origin, as it allows the specification of a
-                custom "upwards" direction (default: 'projection').
+                custom "upwards" direction (default: 'look_at').
             K (torch.Tensor): Camera intrinsics matrix. Note that, unlike
                 standard notation, K here is a 4 x 4 matrix (with the last
                 row and last column drawn from the 4 x 4 identity matrix)
                 (default: None)
             rmat (torch.Tensor): Rotation matrix (again, 4 x 4, as opposed
-                to the usual 3 x 3 convention).
+                to the usual 3 x 3 convention) (default: None).
             tvec (torch.Tensor): Translation vector (3 x 1). Note that the
                 (negative of the) tranlation is applied before rotation,
                 to be consistent with the projective geometry convention
                 of transforming a 3D point X by doing
-                torch.matmul(R.transpose(), X - t) (default: None)
+                torch.matmul(R.transpose(), X - t) (default: None).
             perspective_distort (bool): Whether or not to perform perspective
                 distortion (to simulate field-of-view based distortion effects)
                 (default: True).
+            sigma_val (float): Sharpness of the probability distribution of each triangle
+                (Refer to the paper for more details) (default: 1e-5).
+            dist_func (str): Distance function to use (default: 'euclidean')
+            dist_eps (float): A tiny positive number, for numerically stable distance
+                computation (default: 1e-4).
+            gamma_val (float): Sharpness parameter for the aggregation function (Refer to
+                the paper for more details) (default: 1e-4).
+            aggr_func_rgb (str): Aggregation function to use for the RGB channels
+                (default: 'softmax').
+            aggr_func_alpha (str): Aggregation function to use for the alpha channel
+                (default: 'prod').
+            texture_type (str): Type of textures to use ('surface' vs 'vertex')
+                (default: 'surface').
             viewing_angle (float): Angle at which the object is to be viewed
                 (assumed to be in degrees!) (default: 30.)
-            camera_direction (float): Direction in which the camera is facing
+            viewing_scale (float): Scale at which the object is to be viewed
+                (default: 1).
+            eye (torch.Tensor): Location of the "eye" (of the camera). Used in the `look`
+                and `look_at` modes (default: None).
```

Comment:
I believe the default value is now defined by `self.eye = torch.tensor([0, 0, -(1. / math.tan(math.pi * self.viewing_angle / 180) + 1)], device=device)`